source/extensions/filters/common/ext_authz/check_request_utils.cc

```diff
@@ -106,7 +106,7 @@ void CheckRequestUtils::setHttpRequest(
     envoy::service::auth::v3::AttributeContext::HttpRequest& httpreq, uint64_t stream_id,
     const StreamInfo::StreamInfo& stream_info, const Buffer::Instance* decoding_buffer,
```

Comment:
CI is complaining that stream id param is not used anywhere now.

Reply:
Done